vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -6,6 +6,7 @@ def self.ems_inv_to_hashes(inventory)
 
     def initialize
       @data = {}
+      @data_index = {}
```

Comment:
@Fryguy, I believe all the parsers do it this way ... init the `@data` and `@data_index` in the `initialize` method.